README.md

```diff
@@ -45,25 +45,23 @@ Download and unzip the latest version of Avatar Animator from the [release page]
+| --------------- | --------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------------------------------------------------------------------------------- |
+| Zoom            | Go to `Settings` -> `Video` and select `OBS-Camera` from the `Camera` drop-down menu.                                                         | <img src="./images/zoom.png" alt="Avatar Animator on Zoom: configuration" width="600">                |
+| Skype           | Go to `Settings` -> `Audio & Video` and select `OBS-Camera` from the `Camera` drop-down menu.                                                 | TODO                                                                                                  |
+| Microsoft Teams | Click on your profile picture at the top of Teams, then `Settings` -> `Devices` and select `OBS-Camera` from the `Camera` drop-down menu.     | <img src="./images/ms-teams.png" alt="Avatar Animator on Microsoft Teams: configuration" width="600"> |
+| Google Meet     | Start a meeting, then click `Settings` > `General` and select `OBS-Camera` from the `Camera` drop-down menu.                                  | <img src="./images/google-meet.png" alt="Avatar Animator on Google Meet: configuration" width="600">  |
+| Slack           | Start a call, then click on the gear icon in the top left corner of the call window and select `OBS-Camera` from the `Camera` drop-down menu. | TODO                                                                                                  |
 
 
 ## Configuration
 
-The configuration panel (top right in Avatar Animator; figure below) allows the configuration of cameras, avatars and background colours, and display of debug information. To import new avatars or a background images, drag and drop them into Avatar Animator (note that new avatars will not be added to the `avatar` drop down menu). To create a new avatar, follow the instructions as detailed in [AVATARS.md](AVATARS.md).
-
-<div align="center">
-    <img src="./images/control-panel.png" alt="Avatar Animator control panel" width="200">
-    <br>
-    <b>Avatar Animator control panel.</b>
-</div>
+The configuration panel (top right in Avatar Animator; figure 2) allows the **choice of camera, avatar, background color, and display of debug information**. To **add a new background image**, drag and drop it into Avatar Animator (note that new avatars will not be added to the `avatar` drop down menu). A simplified editor (figure 3) for **modifying existing avatars** is provided where anchor-points and colors can be modified. Currently, when a skeleton point is dragged (only those are modifiable), the position of any name-matching illustration keypoints are set to the same position. For more information on how to **create a new avatar or modify and exiting one**, see [AVATARS.md](AVATARS.md).
```

Comment:
"anchor-points" -> "anchor points"
Also, you use anchor-points in one sentence, and keypoints in another.

Reply:
Changed in https://github.com/letmaik/avatar-animator/pull/29/commits/83eaeefbd3e6767c3005712d1c06d49abe5b3e15